<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetEditor_Sprite</class>
 <widget class="QMainWindow" name="AssetEditor_Sprite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>456</width>
             <height>551</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_properties">
              <property name="title">
               <string>Properties</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_widthTitle">
                   <property name="text">
                    <string>Width:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_widthValue">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_heightLabel">
                   <property name="text">
                    <string>Height:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_heightValue">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="1" rowspan="2" colspan="2">
                <spacer name="horizontalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>276</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="2" rowspan="2">
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>254</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="0" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_numSubimagesLabel">
                   <property name="text">
                    <string>Number of subimages:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_numSubimagesValue">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_OriginLabel">
                 <property name="text">
                  <string>Origin:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" colspan="3">
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_originX">
                   <property name="text">
                    <string>X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_originX"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_originY">
                   <property name="text">
                    <string>Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_originY"/>
                 </item>
                </layout>
               </item>
               <item row="5" column="0" colspan="3">
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <spacer name="horizontalSpacer_9">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_originCenter">
                   <property name="text">
                    <string>Center</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_10">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Collision Checking</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBox_preciseCollisionChecking">
                 <property name="text">
                  <string>&amp;Precise collision checking</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBox_seperateCollisionMasks">
                 <property name="toolTip">
                  <string>Seperate collision masks for subimages</string>
                 </property>
                 <property name="text">
                  <string>Seper&amp;ate collision masks</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_alphaTolerance">
                 <property name="text">
                  <string>Alpha Tolerance:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEdit_alphaTolerance">
                 <property name="maximumSize">
                  <size>
                   <width>64</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QSlider" name="horizontalSlider_alphaTolerance">
                 <property name="maximum">
                  <number>255</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="3">
                <widget class="QGroupBox" name="groupBox_boundingBox">
                 <property name="title">
                  <string>Bounding Box</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_bboxType">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Type:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QComboBox_NoScroll" name="comboBox_bboxType"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_bboxShape">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Shape:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QComboBox_NoScroll" name="comboBox_bboxShape"/>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <layout class="QGridLayout" name="gridLayout_4">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_bboxLeft">
                      <property name="text">
                       <string>Left: </string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="lineEdit_bboxLeft"/>
                    </item>
                    <item row="0" column="2">
                     <widget class="QLabel" name="label_bboxRight">
                      <property name="text">
                       <string>Right:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="3">
                     <widget class="QLineEdit" name="lineEdit_bboxRight"/>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_bboxTop">
                      <property name="text">
                       <string>Top:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLineEdit" name="lineEdit_bboxTop"/>
                    </item>
                    <item row="1" column="2">
                     <widget class="QLabel" name="label_bboxBottom">
                      <property name="text">
                       <string>Bottom:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="3">
                     <widget class="QLineEdit" name="lineEdit_bboxBottom"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_textureSettings">
              <property name="title">
               <string>Texture Settings</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QCheckBox" name="checkBox_tileHorizontal">
                 <property name="text">
                  <string>Tile: Horizontal</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="checkBox_tileVertical">
                 <property name="text">
                  <string>Tile: Vertical</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QCheckBox" name="checkBox_usedFor3D">
                 <property name="text">
                  <string>Used for 3D (Must be a power of 2)</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_textureGroup">
                 <property name="text">
                  <string>Texture Group:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox_NoScroll" name="comboBox_textureGroup"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>17</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_previewAndFrames">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>4</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <widget class="QToolButton" name="toolButton_animationToggle">
                 <property name="toolTip">
                  <string>Play/Pause animation</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="skin.qrc">
                   <normaloff>:/main/icons/control.png</normaloff>
                   <normalon>:/main/icons/control-pause.png</normalon>:/main/icons/control.png</iconset>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::NoArrow</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line_2">
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="midLineWidth">
                  <number>0</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_zoomFit">
                 <property name="toolTip">
                  <string>Fit sprite to view</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="skin.qrc">
                   <normaloff>:/main/icons/magnifier-zoom-fit.png</normaloff>:/main/icons/magnifier-zoom-fit.png</iconset>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_zoomOriginal">
                 <property name="toolTip">
                  <string>Zoom to 100%</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="skin.qrc">
                   <normaloff>:/main/icons/magnifier-zoom-actual-equal.png</normaloff>:/main/icons/magnifier-zoom-actual-equal.png</iconset>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_zoomOut">
                 <property name="toolTip">
                  <string>Zoom out</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="skin.qrc">
                   <normaloff>:/main/icons/magnifier-zoom-out.png</normaloff>:/main/icons/magnifier-zoom-out.png</iconset>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton_zoomIn">
                 <property name="toolTip">
                  <string>Zoom in</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="skin.qrc">
                   <normaloff>:/main/icons/magnifier-zoom-in.png</normaloff>:/main/icons/magnifier-zoom-in.png</iconset>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="toolButton">
                 <property name="toolTip">
                  <string>Toggle transparency background</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="skin.qrc">
                   <normaloff>:/main/icons/checkerboard.png</normaloff>:/main/icons/checkerboard.png</iconset>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="autoRepeatDelay">
                  <number>300</number>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_11">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>103</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGraphicsView" name="graphicsView_spritePreview"/>
            </item>
           </layout>
          </widget>
          <widget class="WidgetSpriteFrameView" name="spriteFrameView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_Close"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionSave_Close">
   <property name="icon">
    <iconset resource="skin.qrc">
     <normaloff>:/main/icons/tick.png</normaloff>:/main/icons/tick.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp; Close</string>
   </property>
   <property name="toolTip">
    <string>Save and close Sprite</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QComboBox_NoScroll</class>
   <extends>QComboBox</extends>
   <header>QComboBox_NoScroll.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetSpriteFrameView</class>
   <extends>QFrame</extends>
   <header>WidgetSpriteFrameView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="skin.qrc"/>
 </resources>
 <connections/>
</ui>
